docs/client/VpnGateway.md

```diff
@@ -0,0 +1,25 @@
+# VpnGateway
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**AvailabilityDomain** | **string** |  | [optional] [default to null]
+**BgpConfig** | [***BgpConfig**](BGPConfig.md) |  | [optional] [default to null]
```

Comment:
Some links like this BgpConfig and the VpnAuthConfig return 404s, but maybe that's because their source has not been merged into develop yet.

Reply:
Submitted PR to fix this.